build-system/single-pass.js

```diff
@@ -521,6 +526,44 @@ function wrapMainBinaries() {
   });
 }
 
+/**
+ * Appends the listed file to the built js binary.
+ * TODO(erwinm, #18811): This operation is needed but straight out breaks
+ * source maps.
+ */
+function postProcessConcat() {
+  const extensions = extensionBundles.filter(
```

Comment:
@choumx do you mind reviewing this part since you did a bit of the same work around in the multi-pass code.

Reply:
Is this meant to subsume `appendToCompiledFile()` in gulpfile.js? I don't see that being removed anywhere.

Reply:
In that case, can we call `appendToCompiledFile()` here? We should at least de-dupe.